Script/build_packages.py

```diff
@@ -75,7 +77,7 @@ def refresh_staging():
     return staging_path
 
 def extract_into(destdir, zippath):
-    with zipfile.ZipFile(zippath, "r") as z:
+    with contextlib.closing(zipfile.ZipFile(zippath, "r")) as z:
```

Comment:
contextlib.closing on zipfiles shouldn't be necessary for recent versions of python. (2.7 and above)